dashboard/src/common/chartComponents/TableChart.tsx

```diff
@@ -2,7 +2,13 @@ import { DataGrid } from "@mui/x-data-grid";
 import { toDate } from "../../../src/utils/index";
 import { percentageFieldList } from "../../constants";
 
-export const TableChart = (_datasetLabel: string, dataTable: any, _dataLength: number) => {
+interface TableChartProps {
+  _datasetLabel: string;
```

Comment:
Can the `_` be removed on these or do they have significance?

Reply:
Good point, ill remove them